pkg/vulnsrc/nvd/nvd.go

```diff
@@ -198,6 +198,7 @@ func (nvd *NVD) Put(tx *bolt.Tx, cve Cve) error {
 		Description:      description,
 		PublishedDate:    &publishedDate,
 		LastModifiedDate: &lastModifiedDate,
+		VulnStatus:       cve.VulnStatus,
```

Comment:
I'm afraid their status is not consistent, like `REJECTED` or `rejected`. I think we should get it aligned on our end.

```suggestion
		VulnStatus:       strings.ToUpper(cve.VulnStatus),
```